src/napari_deeplabcut/_widgets.py

```diff
@@ -641,11 +647,11 @@ def __init__(self, napari_viewer):
         launch_tutorial = QAction("&Launch Tutorial", self)
         launch_tutorial.triggered.connect(self.start_tutorial)
         self.viewer.window.view_menu.addAction(launch_tutorial)
-
+      
         # Add action to view keyboard shortcuts
-        display_shortcuts = QAction("&Shortcuts", self)
-        display_shortcuts.triggered.connect(self.display_shortcuts)
-        self.viewer.window.help_menu.addAction(display_shortcuts)
```

Comment:
Please keep those (especially the help menu); I like the display_shortcuts_action variable name though, we can change it.